content/data/people.yaml

```diff
@@ -0,0 +1,56 @@
+# Generate the gifPlaceHolder by selecting a nice jpg from the gif
+# using this site: https://ezgif.com/gif-to-jpg
+- id: dan
+  fullName: Daniel Burkhardt Cerigo
+  title: Founding Data Scientist
+  bio: Physics & Philosophy, Oxford. Consulted for UK Gov Office of Rail & Road. Chairperson for nonprofit orgs. Academically published. Kaggle ML-competition ranked. Built reinforcement-learning pricing optimisation algo operating on millions (£) of revenue.
+  email: dan@datavaluepeople.com
+  linkedIn: dbcerigo
+  github: DBCerigo
+  gifPlaceHolder: ../../content/assets/people/dan-gif-placeholder.jpg
+  profilePictureGif: ../../content/assets/people/dan.gif
+- id: benjamin
```

Comment:
This file is dope. Nice way to do it all. (Not that I would have expected anything less!)